pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -368,6 +367,7 @@ private static void start() throws MalformedURLException {
         }
 
         post(API.REGISTRATION, new UserRegistrationRoute(interpreter, new TaskPubSubPublisher()), responseSerializer);
+        post(API.USER_SPECIFIC, new UserCreationRoute(), responseSerializer);
```

Comment:
If your intention is to add a new post endpoint at `/user`, I don't think this is going to do what you want it do to. The `USER_SPECIFIC` includes a `:userGuid` path parameter and, if this is creating a user, we do not have a GUID yet.